content/syllabuses/ios-mobile.md

```diff
@@ -6,12 +6,55 @@ title: iOS Mobile Development
 
 [Stanford CS193P iOS11 Swift4](https://drive.google.com/drive/folders/1-TWZDChwwzkiGzt78QlPZDzN-j82JfR6)
 
-## Part 1
+## Swift and More
 
-- Topic : iOS Overview (Lecture 1)
-- Topic : MVC (Lecture 2)
-- Topic : Swift Development and other Tools (Lecture 3)
+- Topic : iOS Overview 
```

Comment:
Should these pages not be created and content linked here

Reply:
Those are all sub-topics of : Swift and More. Swift and more which is the main is a prerequisite to the project Concentration game. All the topics listed are whats covered in the actual topic material and video tutorials.